src/borrow/mod.rs

```diff
@@ -1,4 +1,33 @@
-//! Types and traits for obtaining temporary access to the internals of JavaScript values.
+//! Provides temporary access to JavaScript typed arrays.
+//!
+//! ## Typed Arrays
+//!
+//! JavaScript's [typed arrays](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Typed_arrays)
```

Comment:
FYI, links support anchors which can be useful for limiting the impact to flow within the next. For example:

```
This is a link to [Rust Lang][rust-lang]. It's really cool and you should check it out.

[rust-lang]: https://www.rust-lang.org/
```

This is a link to [Rust Lang][rust-lang]. It's really cool and you should check it out.

[rust-lang]: https://www.rust-lang.org/